python/_trace.py

```diff
@@ -0,0 +1,108 @@
+from enum import IntEnum, Enum
+from ._common import *
+from ._infer import Tensor
+from typing import Iterable, Any, Union
+import abc
+
+
+class TraceLevel(IntEnum):
+    TIMESTAMPS: int = 4
+    TENSORS: int = 8
+
+    def __str__(self) -> str:
+        # TRITONSERVER_InferenceTraceLevelString
+        raise "Not Implemented"
+
+
+# Activity related to timeline tracing
+class TraceActivity(Enum):
```

Comment:
Just an FYI, @Tabrizian is working (IIRC) on custom tracing for python backend, which introduces new `TraceActivity`: https://github.com/triton-inference-server/core/pull/172. So wee need to coordinate this at some point